packages/app/components/Today/QueueCard.tsx

```diff
@@ -67,96 +72,118 @@ const HeaderText = styled.div`
   color: #bfbfbf;
   font-variant: small-caps;
   margin-bottom: 8px;
-`;
+`
 
 const OpenQueueButton = styled(Button)`
-  color: #5f6b79;
   border-radius: 6px;
-  font-weight: 500;
   font-size: 14px;
   margin-left: 16px;
-`;
+`
 
 const EditNotesButton = styled(Button)`
   border-radius: 6px;
   font-size: 14px;
   font-weight: 500;
-`;
+`
 
 const SaveButton = styled(Button)`
   background: #2a9187;
   border-radius: 6px;
   color: white;
   font-weight: 500;
   font-size: 14px;
-`;
+`
 
 const NotesDiv = styled.div`
   display: flex;
   flex-direction: row;
   flex-grow: 1;
   margin: 10px 10px auto 0;
-`;
+`
 
 const RightQueueNotesRow = styled.div`
   display: flex;
-`;
+`
 const NotesInput = styled(Input.TextArea)`
   border-radius: 6px;
   border: 1px solid #b8c4ce;
   //display: flex;
   //flex-grow: 1;
   margin: auto 0px auto 10px;
-`;
+`
 
 const Notes = styled.div`
   overflow-wrap: break-word;
   white-space: pre-wrap;
-`;
+`
 
 const StyledKOHAvatar = styled(KOHAvatar)`
-  margin-right: 25px;
-  margin-top: 10px;
-`;
+  margin-right: 1rem;
+`
 
 const QueueCardButtonRow = styled(Row)`
   padding-top: 10px;
-`;
+`
 
 const QueueCardDivider = styled(Divider)`
   margin-top: 12px;
   margin-bottom: 0;
-`;
+`
 
 const NotesSkeleton = styled(Skeleton)`
   width: 60%;
-`;
+`
 
 const QueueCard = ({
   queue,
   isTA,
-  updateQueueNotes
+  updateQueueNotes,
 }: QueueCard): ReactElement => {
-  const [editingNotes, setEditingNotes] = useState(false);
-  const [updatedNotes, setUpdatedNotes] = useState(queue.notes);
-  const router = useRouter();
-  const { cid } = router.query;
+  const [editingNotes, setEditingNotes] = useState(false)
+  const [updatedNotes, setUpdatedNotes] = useState(queue.notes)
+  const router = useRouter()
+  const { cid } = router.query
 
-  const staffList = queue.staffList;
+  const staffList = queue.staffList
 
   const handleUpdate = () => {
-    setEditingNotes(false);
-    updateQueueNotes(queue, updatedNotes);
-  };
+    setEditingNotes(false)
+    updateQueueNotes(queue, updatedNotes)
+  }
   return (
     <PaddedCard
```

Comment:
What if we made the whole card clickable? Just an idea.

Reply:
Ooh good idea! That would certainly make it easier to navigate to the queue since they wouldn't need to find the button. Plus removing the button would mean less visual clutter, which is always a good thing. I shall investigate this later to see if it would break anything